plugins/module_utils/podman/podman_pod_lib.py

```diff
@@ -538,6 +538,7 @@ def __init__(self, module, name, module_params):
         self.name = name
         self.stdout, self.stderr = '', ''
         self.info = self.get_info()
+        self.ps_info = self.get_ps()
```

Comment:
I would avoid executing it on all version, but only when it's needed. For example see below